src/mat/4C_mat_micromaterial.hpp

```diff
@@ -182,6 +198,12 @@ namespace Mat
     std::string micro_input_file_name() const { return params_->microfile_; }
     int micro_dis_num() const { return params_->microdisnum_; }
     double init_vol() const { return params_->initvol_; }
+
+    [[nodiscard]] PAR::MicroMaterial::RuntimeOutputOption runtime_output_option() const
```

Comment:
If you give up the splitting of the decision (`none`, `gp1`, `all`) as written in my other comment, this method can be deleted because this information does not need to be exposed to the multi scale model evaluator (`4C_structure_new_model_evaluator_multiscale.cpp`).